apps/rush/src/actions/VersionAction.ts

```diff
@@ -119,12 +121,25 @@ export default class VersionAction extends BaseRushAction {
     }
   }
 
+  private _validateResult(): void {
+    // Load the config from file to avoid using inconsistent in-memory data.
+    const rushConfig: RushConfiguration =
+      RushConfiguration.loadFromConfigurationFile(this.rushConfiguration.rushJsonFile);
+    const mismatchFinder: VersionMismatchFinder = new VersionMismatchFinder(rushConfig.projects);
+    if (mismatchFinder.numberOfMismatches) {
+      throw new Error('Mismatched dependencies are found. Run \'rush check\' to find more details.');
```

Comment:
> \'rush  [](start = 61, length = 8)

English uses double-quotes

Reply:
Possible wording improvement:  "Unable to update the versions because inconsistencies were encountered. Run "rush check" for details."

---
In reply to: [145836199](https://github.com/Microsoft/web-build-tools/pull/400#discussion_r145836199) [](ancestors = 145836199)